doc/audit/builtin_maps.rst

```diff
@@ -108,6 +108,19 @@ Identity, status and attestations (endorsed quotes) of the nodes hosting the net
 .. doxygenenum:: ccf::QuoteFormat
    :project: CCF
 
+``network_configurations``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The currently valid and in-flight network configurations of the network.
+
+**Key** Reconfiguration ID: a unique identifier of a configuration
+
+**Value** A set of node IDs of the nodes in the respective configuration. Represented as JSON.
```

Comment:
JSON doesn't have sets, so it would be good to mention whether that's represented as an array or as a object with special values.